charter/index.html

```diff
@@ -167,40 +172,41 @@ <h1 id="title">PROPOSED Solid Working Group Charter</h1>
+        <span class=new>It has regularly sought cross-fertilization with the aforementioned communities, among others (e.g., the <a href="https://www.w3.org/community/fed-id/">Federated Identity CG</a> and the <a href="https://www.w3.org/community/socialcg/">Social Web Incubator CG</a>).</span></p>
 
-        <p>The <cite><a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a></cite> has been incubating <a href="https://solidproject.org/TR/">Technical Reports</a> since 2018 towards that end: <q cite="https://www.w3.org/groups/cg/solid">to describe the interoperability between different classes of products by using web communication protocols, global identifiers, authentication and authorization mechanisms, data formats and shapes, notifications, and query interfaces.</q></p>
+        <p>The broader Solid community has developed a <a href="https://solidproject.org/developers/tools/">suite of implementations, tools, and libraries</a> for developers, as well as <a href="https://solidproject.org/apps">applications</a> for users.
+        <span class=new>See a separate <a href="https://timbl.solidcommunity.net/2023/03%20EWADA/TAGExplainerforSolid.html">explainer</a> for more detailed description of the Solid approach.</span>
+        </p>
 
-        <p>The wider Solid community has developed a <a href="https://solidproject.org/developers/tools/">suite of implementations, tools, and libraries</a> for developers, as well as <a href="https://solidproject.org/apps">applications</a> for users.</p>
       </div>
 
       <section id="scope" class="scope">
         <h2>Scope</h2>
+
         <p>
-                The design approach for the <a href="https://solidproject.org/TR/protocol">Solid Protocol</a> will continue substantial efforts undertaken previously in the <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a> to specify a set of interoperable protocol standards for enabling the use and development of Solid servers and applications across a <a href="https://github.com/solid/user-stories">wide range of use cases</a>.
+          <span class=new>The scope of the Working Group is to define a web protocol for use between client-side applications, on the one hand, and storage and identity servers, on the other, allowing</span> users to keep authority over their data, identity, and privacy.
+          This protocol will ensure interoperability between compliant user applications, and compliant storage and identity servers,
+          thus giving users the freedom to select the applications and services that suit their needs.
         </p>
 
+
         <p>
-                The Working Group will:
+        Based on the <a href="https://github.com/solid/user-stories">user cases</a> identified by the Solid project, potentially extended by Working Group participants, the Working Group will:
```

Comment:
```suggestion
        Based on the <a href="https://github.com/solid/user-stories">user stories</a> identified by the Solid project, potentially extended by Working Group participants, the Working Group will:
```

These are specifically not use case, they are *user stories* and should be understood as such. They haven't been curated the way they were intended, but it is vitally important to have a low-threshold input mechanism for people who are not deeply that can also be useful to the development process. And that's pretty hard to achieve.

Asking people to submit use cases is high threshold, and also quite a burden on people. Writing mailing list or forum posts is nice to get feedback and incubate an idea, but both is high threshold and may often be too much to read for developers, and so does not meet the purpose.

So, yes, the curation there hasn't happened and it should. Once a story can be done, the issue should be closed with an explanation (like a pointer to documentation) on how. But the big point remains, the general public needs a way to communicate what they could use into the project. The WG could of course design a different mechanism for that, but it has been a largely neglected part of standards development that has to change.